.discordenorc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
should this be in the repo? also i thought we wanted to go with discordeno.config.js file name?